app/scripts/models/reliers/fx-desktop.js

```diff
@@ -78,6 +80,10 @@ define([
      */
     isCustomizeSyncChecked: function () {
       return !! (this.isSync() && this.get('customizeSync'));
+    },
+
+    isSignupDisabled: function () {
+      return this.get('exclude_signup') === '1';
     }
```

Comment:
We could add an error, something like `FORCE_SIGNUP_DISABLED` or `SIGNUP_DISABLED_VIA_URL` with an error message like `Signup has been disabled.`